tests/PHPUnit/Integration/Tracker/VisitTest.php

```diff
@@ -414,11 +414,12 @@ public function test_markArchivedReportsAsInvalidIfArchiveAlreadyFinished_Should
 
     public function test_markArchivedReportsAsInvalidIfArchiveAlreadyFinished_shouldConsiderWebsitesTimezone()
     {
-        $timezone1 = 'UTC+4';
-        $timezone2 = 'UTC+6';
+        // The double-handling below is needed to work around weird behaviour when UTC and UTC+5 are different dates
+        // Example: 4:32am on 1 April in UTC+5 is 11:32pm on 31 March in UTC
+        $midnight = Date::factoryInTimezone('today', 'UTC+5')->setTimezone('UTC+5');
 
-        $currentActionTime1 = Date::today()->setTimezone($timezone1)->getDatetime();
-        $currentActionTime2 = Date::today()->setTimezone($timezone2)->getDatetime();
+        $oneHourAfterMidnight = $midnight->addHour(1)->getDatetime();
```

Comment:
those variables are a lot more easier to read and make a lot more sense 👍 This way I would have probably noticed while writing the tests the previous assertions were wrong :)